### PR TITLE
Rename hookshot to rectangle-pro and update from 1.23.1,54 to 2.0,100

### PR DESCRIPTION
**Pull request by Aaron-Rumpler**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/rectangle-pro.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  uninstall quit: "com.knollsoft.Hookshot"
+
+  zap trash: [
+    "~/Library/Application Support/Rectangle Pro",
+    "~/Library/Caches/com.knollsoft.Hookshot",
+    "~/Library/Cookies/com.knollsoft.Hookshot",
+    "~/Library/Cookies/com.knollsoft.Hookshot.*",
+    "~/Library/HTTPStorages/com.knollsoft.Hookshot",
+    "~/Library/HTTPStorages/com.knollsoft.Hookshot.*",
```
Comment (neutric) [inline Casks/rectangle-pro.rb:28]: Same comment as in https://github.com/Homebrew/homebrew-cask/pull/117446 regarding use of `*`.

**Comment by miccal** [comment]
Please do not remove the `hookshot` Cask just yet:
```
==> Analytics
install: 29 (30 days), 103 (90 days), 332 (365 days)
```
Instead, please add the following to the end of the `hookshot` cask:
```
  caveats do
    discontinued
  end
```

**Comment by miccal** [comment]
@Aaron-Rumpler and make it snappy please, I want to install `rectangle-pro`!

(Joking, of course 😄)

**Comment by miccal** [comment]
Yay!

**Pull request merged**